Gruntfile.js

```diff
@@ -28,42 +22,31 @@ grunt.initConfig( {
 	concat: {
 		options: { process: process },
 		"src-js": {
-			src: [
-				"src/intro.js",
-				"src/core/initialize.js",
-				"src/core/utilities.js",
-				"src/core/stacktrace.js",
-				"src/core/config.js",
-				"src/core/logging.js",
-				"src/core/onerror.js",
-				"src/core.js",
-				"src/test.js",
-				"src/assert.js",
-				"src/equiv.js",
-				"src/dump.js",
-				"src/export.js",
-				"src/outro.js",
-				"runner/fixture.js",
-				"reporter/intro.js",
-				"reporter/urlparams.js",
-				"reporter/html.js",
-				"reporter/diff.js",
-				"reporter/outro.js"
-			],
+			src: "dist/qunit.js",
 			dest: "dist/qunit.js"
```

Comment:
This is correct. I'll look into a replacement, but since we need to do it for both `qunit.js` and `qunit.css` I don't think this is too bad if we wind up leaving it.